edx_rest_framework_extensions/tests/test_settings.py

```diff
@@ -23,3 +27,28 @@ def test_get_setting(self):
         with override_settings(EDX_DRF_EXTENSIONS=_settings):
             for key, value in _settings.items():
                 self.assertEqual(get_setting(key), value)
+
+    def test_get_jwt_issuers(self):
+        """
+        Verify the get_jwt_issuers operation returns the proper information
+        """
+        _jwt_issuers = deepcopy(settings.JWT_ISSUERS)
+        _deprecated = [
+            {
+                'ISSUER': settings.JWT_AUTH['JWT_ISSUER'],
+                'SECRET_KEY': settings.JWT_AUTH['JWT_SECRET_KEY'],
+                'AUDIENCE': settings.JWT_AUTH['JWT_AUDIENCE'],
+            }
+        ]
+
+        # Test deprecated settings
+        with mock.patch('edx_rest_framework_extensions.settings.logger') as patched_log:
+            with override_settings(JWT_ISSUERS=None):
+                self.assertEqual(get_jwt_issuers(), _deprecated)
+
+            msg = "'JWT_ISSUERS' list not defined, checking for deprecated settings.'"
+            patched_log.info.assert_any_call(msg)
+
+        # Test current settings
+        with override_settings(JWT_ISSUERS=_jwt_issuers):
```

Comment:
This is essentially calling `settings.JWT_ISSUERS = settings.JWT_ISSUERS`. Unless you are going to actually set a different value or modify `settings.JWT_ISSUERS`, you don't need to call `override_settings`.